sdk/digitaltwins/dtdl-parser/README.md

```diff
@@ -53,24 +53,36 @@ To parse a sample DTDL model, either start with one you have already written or
 
 ```js
 // example.js
+const { ModelsRepositoryClient } = require("@azure/iot-modelsrepository");
+const { ModelParsingOption, createParser } = require("@azure/dtdl-parser");
+
 async function main() {
-  const client = new ModelsRepositoryClient();
   const dtmi = "dtmi:com:example:TemperatureController;1";
-  const models = await client.getModels(dtmi);
 
+  const repositoryClient = new ModelsRepositoryClient();
   const modelParser = createParser(ModelParsingOption.PermitAnyTopLevelElement);
-  modelParser.options = ModelParsingOption.PermitAnyTopLevelElement;
-  Object.entries(models).forEach(([key, value]) => {
-    console.log(`dtmi: ${key}`);
-    const modelDict = await modelParser.parse([value]);
-    Object.entries(modelDict).forEach(([key2, value2]) => {
-      console.log(key2);
-    });
-  });
+
+  // Bind the model repository to the parser so the parser can use it to load
+  // any dependant models.
```

Comment:
>/ any dependant models. [](http://example.com/codeflow?start=3&length=23)

Please review the comments in this example for correct use of terms (specifically "dependant models" and "the contents of each model")